code/modules/jobs/job_types/ncr.dm

```diff
@@ -982,7 +982,7 @@ Access
 	head = /obj/item/clothing/head/f13/ncr/storm
 	suit = /obj/item/clothing/suit/armor/medium/vest/ncr/mant
 	backpack_contents = list(
-		/obj/item/melee/onehanded/knife/bayonet = 1,
+		/obj/item/shovel/trench = 1
```

Comment:
```suggestion
		/obj/item/shovel/trench = 1,
```

Reply:
These are my bad,I should have double checked before I did the pull request